packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -770,8 +770,8 @@ test.serial('writeGranuleFromApi() throws for a granule with an invalid collecti
   } = t.context;
 
   await t.throwsAsync(
-    writeGranuleFromApi({ ...granule, collectionId: 'wrong___collection' }, knex),
-    { message: 'Record in collections with identifiers {"name":"wrong","version":"collection"} does not exist.' }
+    writeGranuleFromApi({ ...granule, collectionId: constructCollectionId('wrong____', 'collection') }, knex),
```

Comment:
Are they not allowed the underscores? What is this testing?

Reply:
Or what is invalid about that collectionId ?

Reply:
I was just adding spice to an existing test.  It's wrong because it's not in the database.